plugins/auth-backend/src/lib/PassportStrategyHelper.ts

```diff
@@ -149,12 +148,12 @@ export const executeRefreshTokenStrategy = async (
 };
 
 export const executeFetchUserProfileStrategy = async (
-  providerstrategy: passport.Strategy,
+  providerStrategy: passport.Strategy,
   accessToken: string,
   params: any,
 ): Promise<ProfileInfo> => {
   return new Promise((resolve, reject) => {
-    const anyStrategy = providerstrategy as any;
+    const anyStrategy = providerStrategy as any;
```

Comment:
Not related to this PR, but it would prolly be best to define our own type here instead of using any